doc/user-docs/UsingJsonSettings.md

```diff
@@ -126,3 +126,47 @@ More information about UWP URI schemes [here](https://docs.microsoft.com/en-us/w
     1. URL such as
 `http://open.esa.int/files/2017/03/Mayer_and_Bond_craters_seen_by_SMART-1-350x346.jpg` 
     2. Local file location such as `C:\Users\Public\Pictures\openlogo.jpg`
+
+### Adding Copy and Paste Keybindings 
+
+As of [#1093](https://github.com/microsoft/terminal/pull/1093) (first available in Windows Terminal v0.3), the Windows Terminal now
+supports copy and paste keyboard shortcuts. However, if you installed and ran
+the terminal before that, you won't automatically get the new keybindings added
+to your settings. If you'd like to add shortcuts for copy and paste, you can do so by inserting the following objects into your `globals.keybindings` array:
+
+```json
+{ "command": "copy", "keys": ["ctrl+shift+c"] },
+{ "command": "paste", "keys": ["ctrl+shift+v"] }
+```
+
+This will add copy and paste on <kbd>ctrl</kbd>+<kbd>shift</kbd>+<kbd>c</kbd>
```

Comment:
personal preference: i like the look of <kbd>Ctrl+Shift+X</kbd> over <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>X</kbd>. It looks like a single cohesive _thing_ the user can do.